ops/services/alerts/app_service_metrics/main.tf

```diff
@@ -1,6 +1,10 @@
+locals {
+  ucfirst_env = title(var.env)
```

Comment:
Can we rename this to something more verbose?

Reply:
Can do—how would `env_title` grab you? Since I assume by "more verbose" you really mean "more understandable for people who don't have perl functions as part of their working memory." 😁

Reply:
Sure, that works. I _just now_ realized that "uc" was for "uppercase", so I was very confused earlier.